README.md

```diff
@@ -65,7 +64,18 @@ $ yarn workspace <package-name> test
 
 # For example for "code-du-travail-frontend"
 $ yarn workspace code-du-travail-frontend test
+```
+
+### Prod
+
+Edit `.env`
```

Comment:
I would say 

```sh 
$ cp .env.sample .env
```
And edit the .env file.

Reply:
Isn't is automatique on "postinstall" ?